hs_core/serialization.py

```diff
@@ -718,32 +716,19 @@ def write_metadata_to_resource(self, resource):
                           'phone': c.phone, 'homepage': c.homepage,
                           'researcherID': c.researcherID,
                           'researchGateID': c.researchGateID}
-                try:
-                    resource.metadata.create_element('contributor', **kwargs)
-                except IntegrityError:
-                    pass
+                resource.metadata.create_element('contributor', **kwargs)
             else:
                 msg = "Contributor with type {0} are not supported"
                 msg = msg.format(c.__class__.__name__)
                 raise TypeError(msg)
         if self.abstract:
-            try:
-                resource.metadata.create_element('description', abstract=self.abstract)
-            except IntegrityError:
-                resource.metadata.update_element('description', resource.metadata.description.id,
-                                                 abstract=self.abstract)
+            resource.metadata.create_element('description', abstract=self.abstract)
```

Comment:
@selimnairb I guess the reason you made this change is because it is impossible for 'description' element to exist already in this case so no need to do update_element() call?